config/settings.local.yaml.tpl

```diff
@@ -1,7 +1,6 @@
 #default:
 #  skip_cleanup: false
 #  tester: "someuser"                          # Optional: name of the user, who is running the tests, defaults to whoami/uid
-#  standalone: false                           # True, if Testsuite should test only individual components (e.g. Authorino/limitador operators)
```

Comment:
I'd like to also have a standalone option in the settings file, at least for now.

Reply:
Why?

Reply:
Well, I guess it doesn't matter now as we started using the pytest marks anyway. But I don't like that I need to change my code editor settings for every test that I'm running. With the settings file this change is only a true/false, without - I should remember every flag every time I change my testing scope. But from a full testsuite run through makefile perspective, it is of course better to have every option as a flag. I hope there are some presets in Pycharm for this, I probably can get used to it. 

Can't settings file just contain an option preset and flag is always overriding?

Reply:
With the settings, it is too easy to mistakenly leave it on and think it is running Kuadrant with Gateway API and instead it can be standalone, you do not have an easy way of telling and all would be green anyway. This makes it much more explicit and doesn't allow mistakes.